baybe/recommenders/pure/bayesian/base.py

```diff
@@ -20,7 +21,9 @@
 class BayesianRecommender(PureRecommender, ABC):
     """An abstract class for Bayesian Recommenders."""
 
-    surrogate_model: SurrogateProtocol = field(factory=GaussianProcessSurrogate)
+    _surrogate_model: SurrogateProtocol = field(
```

Comment:
@Scienfitz, @AVHopp: Now that we have the chance to rename for free if we want: do you prefer `surrogate_model` or just `surrogate`?

Reply:
I prefer `surrogate_model`.

Reply:
indifferent

Reply:
after more time i also prefer the model version